OWNERS

```diff
@@ -4,7 +4,15 @@ approvers:
   - ingress-nginx-admins
   - ingress-nginx-maintainers
   - ElvinEfendi
+  - whalecold
```

Comment:
Only need to add core maintainers into approvers.

Reply:
Done.